### PR TITLE
Adding verifier smart contract package

### PR DESCRIPTION
**Pull request by Meyanis95**
## Motivation

This PR add a new contract package that will enable users to directly import the on-chain verifier in their project.

## Change Summary

- Added contract package with 1 relevant test
- Fixed version management command of the monorepo
- Fixed a bug in kit causing infinite status when proof not verified
- Fixed the TS config in pcd-country-identity
- Added scripts in pcd-country-identity to generate contracts

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [ ] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [change set](https://github.com/privacy-scaling-explorations/anon-aadhaar/CHANGELOG.md)
- [ ] PR has been tagged with a change label(s) (i.e. documentation, feature, bug fix, or chore)
- [ ] PR includes documentation if necessary.

## Additional Context

If this is a relatively large or complex change, provide more details here that will help reviewers

### CODE REVIEW

File: packages/country-identity-contracts/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Sample Hardhat Project
```
Comment (vuvoth) [inline packages/country-identity-contracts/README.md:1]:
```suggestion
# Anon-aadhaar verifier contract
```

File: packages/pcd-country-identity/src/pcd.ts [hunk 2]
```diff
@@ -85,7 +85,9 @@ export async function verify(
   let vk
   if (isWebEnv === true) {
     vk = await axios
-      .get('https://d3dxq5smiosdl4.cloudfront.net/verification_key.json')
+      .get(
+        'https://anon-aadhaar-pcd.s3.eu-west-3.amazonaws.com/verification_key.json'
```
Comment (vuvoth) [inline packages/pcd-country-identity/src/pcd.ts:89]: This should be constant I think

Comment (Meyanis95) [reply packages/pcd-country-identity/src/pcd.ts:89]: You're right, that's a mistake

File: packages/country-identity-contracts/README.md (hunk 1 above)
Comment (Meyanis95) [reply packages/country-identity-contracts/README.md:1]: Good point!

**Review by vuvoth — APPROVED** [review]

**Pull request merged**